docs/release-notes-pxc-v8.0.31.md

```diff
@@ -0,0 +1,63 @@
+    | SETUP_INSTRUMENTS                                                                    | FLAGS                                                     |
+    | THREADS                                                                              | CONTROLLED_MEMORY, MAX_CONTROLLED_MEMORY, TOTAL_MEMORY, MAX_TOTAL_MEMORY |
+    | EVENTS_STATEMENTS_CURRENT, EVENTS_STATEMENTS_HISTORY, EVENTS_STATEMENTS_HISTORY_LONG | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+    | Statement Summary Tables                                                             | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+    | Performance Schema Connection Tables                                                 | MAX_SESSION_CONTROLLED_MEMORY, MAX_SESSION_TOTAL_MEMORY   |
+    | PREPARED_STATEMENTS_INSTANCES                                                        | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+
+    The following columns have been added to the sys schema `STATEMENT_ANALYSIS` and `X$STATEMENT_ANALYSIS` views:
+
+    * MAX_CONTROLLED_MEMORY
+
+    * MAX_TOTAL_MEMORY
+
+    The `controlled_by_default` flag has been added to the `PROPERTIES` column of the `SETUP_INSTRUMENTS` table.
+
+    Now, you can add and remove non-global memory instruments to the set of controlled-memory instruments. To do this, set the value of the `FLAGS` column of `SETUP_INSTRUMENTS`.
+
+    ```sql
+    SQL> UPDATE PERFORMANCE_SCHEMA.SETUP_INTRUMENTS SET FLAGS="controlled" 
+    WHERE NAME='memory/sql/NET::buff';
+    ```
+
+* The `audit_log_flush` variable has been deprecated and will be removed in future releases.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.31 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-31.html).
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona Server for MySQL-based variation of the Percona Distribution for MySQL:
+
+| Component               | Version   | Description                                |
+| ----------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup      | [8.0.31-24](https://docs.percona.com/percona-xtrabackup/8.0/release-notes/8.0/8.0.31-24.0.html)| An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy                 | [2.5.12](https://git.haproxy.org/?p=haproxy-2.5.git;a=commit;h=7367a7ceb9494a4c463120bb065d619c8663b00a) | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL                | [2.4.7](https://docs.percona.com/proxysql/2.4.7.html)| A high performance, high-availability, protocol-aware proxy for MySQL.          |
```

Comment:
Can change if we release  ProxySQL 2.4.8 before release date.

Reply:
Updated